docs/sources/tempo/operations/best-practices.md

```diff
@@ -16,7 +16,7 @@ Spans also have a hierarchy, where parent spans can have children or siblings.
 
 In the screenshot below, the left side of the screen (1) shows the list of results for the query. The right side (2) lists each span that makes up the selected trace.
 
-![Traces example with query results and spans](/static/img/docs/tempo/trace-explore-spans.png)
+![Traces example with query results and spans](/static/img/docs/tempo/screenshot-trace-explore-spans-g10.png)
```

Comment:
since this image does not exist in this repo are there any CI checks that alert us if this is removed?

cc @jdbaldry

Reply:
Very fair question.

The asset uploader service doesn't allow deletes except by the website squad themselves so removals don't happen often. That being said, that distance could mean that it would be hard to communicate if a removal were to happen.

There's nothing turnkey right now but I could imagine a quick script that runs `make docs`, greps for all image links, and then curls the URL to check it doesn't 404.

Reply:
cool, agree that something here would be nice. perhaps some CI in the main repo that double checks links/images aren't broken when they cross sub-repos.

i believe hugo will already catch in-repo broken links.